dlrs/README.md

```diff
@@ -10,9 +10,8 @@ The Deep Learning Reference Stack, is an integrated, highly-performant open sour
 
 This open source community release is part of an effort to ensure AI developers have easy access to all features and functionality of Intel platforms.To offer more flexibility, there are multiple versions of the Deep Learning Reference Stack:
 
-* Intel® Deep Neural Networks Library (Intel® DNNL) primitives and AVX-512 Deep Learning Boost (formerly known as AVX-512 VNNI)  
-* TensorFlow with Eigen optimized for Intel Architecture  
-* TensorFlow with Intel® Deep Neural Networks Library (Intel® DNNL) primitives
+* Tensorflow 2.0 and 1.15 with Intel® Deep Neural Networks Library (Intel® DNNL) primitives and AVX-512 Deep Learning Boost (formerly known as AVX-512 VNNI) 
```

Comment:
Let's add openvino and transformers info also here: 
- [OpenVINO™ - Deep Learning Deployment Toolkit v2019_R3](https://github.com/opencv/dldt/releases/tag/2019_R3)
- [Transformers - State-of-the-art Natural Language Processing for TensorFlow 2.0](https://github.com/huggingface/transformers)

Reply:
Agree on OpenVINO, but perhaps transformers are better on the releasenotes.md file?

Reply:
+1 , yeah that makes more sense.